APIManager.js

```diff
@@ -0,0 +1,95 @@
+//This is the class that will manage all your APIs
+
+class APIManager {
+    constructor() {
+        this.data = {
+            mainUser: {},
+            friends: [],
+            pokemon: {},
+            quote: {},
+            meatQuote: ''
+
+        }
+    }
+
+    _getMeatQuote() {
+
+        $.ajax({
+            method: 'GET',
+            url: 'https://baconipsum.com/api/?type=meat-and-filler',
+            success: (response) => {
+                this.data.meatQuote = response[Math.floor(Math.random() * 5)]
+                console.log('meat quote is loaded')
+            },
+            error: () => console.log('error loading meat quote'),
+        })
+
+    }
+
```

Comment:
Try not to have too many empty lines - try to keep your code tight and use empty lines only when necessary